xml/cap_admin_upgrade.xml

```diff
@@ -131,51 +131,596 @@
    <xref linkend="sec-cap-skipped-release"/>.
   </para>
 
-<screen>&prompt.user;helm list</screen>
+  <para>
+   For upgrades from &productname; 1.5 to 1.5.1, if
+   the <literal>mysql</literal> roles of <literal>uaa</literal> and
+   <literal>scf</literal> are in high availability mode, it is recommended to
+   first scale them to single availability. Performing an upgrade with these
+   roles in HA mode runs the risk of encountering potential database migration
+   failures, which will lead to pods not starting.
+  </para>
 
   <para>
-   Begin by performing an upgrade of <literal>uaa</literal>.
+   The following sections outline the upgrade process for &productname;
+   deployments described by the given scenarios:
   </para>
 
-<screen>&prompt.user;helm upgrade <replaceable>susecf-uaa</replaceable> suse/uaa \
+  <itemizedlist>
+   <listitem>
+    <para>
+     The <literal>mysql</literal> roles are in high availability mode by setting
+     <literal>config.HA</literal> to <literal>true</literal>, see
+     <xref linkend="sec-cap-upgrade-config-ha"/>.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     The <literal>mysql</literal> roles are in high availability mode by
+     configuring custom sizing values, see
```

Comment:
Same as above...

Reply:
Update wording in https://github.com/SUSE/doc-cap/pull/650/commits/bb8821002c60a20ac3e176003e9367c5c9f6f49c